src/main/resources/default/templates/jobs/params/virtual-object-autocomplete-script.html.pasta

```diff
@@ -0,0 +1,50 @@
+<script>
```

Comment:
can't we include this in the parameter page directly?

Reply:
we include this only once. If we move this to the parameter page this would be included for every file parameter